Mathlib/Order/RelSeries.lean

```diff
@@ -652,6 +664,30 @@ lemma Rel.finiteDimensional_or_infiniteDimensional [Nonempty α] :
   rw [← not_finiteDimensional_iff]
   exact em r.FiniteDimensional
 
+variable {r} in
+lemma Rel.finiteDimensional_swap_iff :
```

Comment:
The name suggests that the `iff` is reversed compared to the actual statement, so I think either the statement or the name should change.

Also, how about providing implication variants `FiniteDimensional.swap` and `FiniteDimensional.of_swap`? The former could even be an instance.

Reply:
I think it's best to change the statement to `FiniteDimensional (Function.swap r) ↔ FiniteDimensional r` (in this way it matches the theorem name, moreover I think it could be a `simp` lemma).